rules-java/api/src/main/java/org/jboss/windup/rules/apps/mavenize/pom.xml.ftl

```diff
@@ -0,0 +1,187 @@
+<#-- This file should be kept aligned with JBoss EAP Quickstarts. -->
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <#-- These constructs are intended for correct formatting of the resulting pom.xml. -->
+    <#assign i1 = ""?left_pad(1*4)>
+    <#assign i2 = ""?left_pad(2*4)>
+    <#assign i3 = ""?left_pad(3*4)>
+    <#assign i4 = ""?left_pad(4*4)>
+
+    <#if pom.parent??>
+    <parent>
+        <groupId>${pom.parent.coords.groupId}</groupId>
+        <artifactId>${pom.parent.coords.artifactId}</artifactId>
+        <version>${pom.parent.coords.version}</version>
+    </parent>
+    </#if>
+
+    <groupId>${pom.coords.groupId}</groupId>
+    <artifactId>${pom.coords.artifactId}</artifactId>
+    <#if pom.coords.version??>${i1}<version>${pom.coords.version}</version>${"\n"}</#if><#t><#-- Null if same as parent. -->
+    <#if pom.coords.packaging??>${i1}<packaging>${pom.coords.packaging}</packaging>${"\n"}</#if><#t>
+    <#if pom.coords.classifier??>${i1}<classifier>${pom.coords.classifier}</classifier>${"\n"}</#if><#t>
+
+    <#if pom.name??>${i1}<name>${pom.name}</name>${"\n"}</#if><#t>
+    <#if pom.description??>${i1}<description>${pom.description}</description>${"\n"}</#if><#t>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <!-- maven-compiler-plugin -->
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
I'd suggest just defaulting to 1.7 for now. But I agree that this isn't particularly important for the typical use case of these POMs. Some hand tweaking is expected.